yapftests/yapf_test.py

```diff
@@ -15,6 +15,7 @@
 """Tests for yapf.yapf."""
 
 import io
+from re import match
```

Comment:
Unless you import a lot of things from module it's useless and just a waste of time and space in lines. So without a good reason I surely won't do that.